httpx/client.py

```diff
@@ -162,6 +175,15 @@ def merge_headers(
             return merged_headers
         return headers
 
+    def merge_query_params(
```

Comment:
It seems there's more instances in the code of `queryparam` vs `query_param`, the Issue suggested to use `merge_query_params` -> but I don't see the similar methods being used elsewhere besides in `build_request`, I'm going to make a judgement call and change my `query_params` to `queryparams` to match the rest of the codebase.